This patch is useed as a temporary fix to allow Kyber-JS tu be correctly loaded by NativeScript.
To execute it simple run `patch -p0 < nodeify_temporary_patch.patch` from the root folder of the app.

--- node_modules/nativescript-nodeify/patch-npm-packages.js	2018-02-22 12:26:28.000000000 +0100
+++ node_modules/nativescript-nodeify/patch-npm-packages.js	2018-02-22 11:44:27.000000000 +0100
@@ -71,7 +71,7 @@
     var patched = false;
     if (main && browser) {
       var mainReplacement = typeof browser === "object" ? browser[main] : browser;
-      if (mainReplacement) {
+      if (mainReplacement && !file.name.startsWith('@dedis')) {
         log("\n" + fileName + "'s main ('" + main + "') was replaced by its browser version ('" + mainReplacement + "')");
         file.main = mainReplacement;
         patched = true;
@@ -210,7 +210,7 @@
       var folder = fs.readdirSync(folderStr);
       for (var j = 0; j < folder.length; j++) {
         var folderName = folder[j];
-        if (/*appDependencies.hasOwnProperty(folderName) &&*/ blacklist.indexOf(folderName) === -1 && !folderName.startsWith("@") && (folderName === "nativescript-nodeify" || !folderName.startsWith("nativescript"))) {
+        if (/*appDependencies.hasOwnProperty(folderName) &&*/ blacklist.indexOf(folderName) === -1 && (folderName === "nativescript-nodeify" || !folderName.startsWith("nativescript"))) {
           var f = path.join(folderStr, folderName);
           findFilesByName(f, "package.json", packages);
         }
